modules/020-deckhouse/hooks/check_deckhouse_release.go

```diff
@@ -175,6 +180,59 @@ releaseLoop:
 	return nil
 }
 
+func (dcr *DeckhouseReleaseChecker) generateChangelogForEnabledModules(input *go_hook.HookInput) map[string]interface{} {
+	enabledModules := input.Values.Get("global.enabledModules").Array()
+	enabledModulesChangelog := make(map[string]interface{})
+
+	for _, enabledModule := range enabledModules {
+		if v, ok := dcr.releaseMetadata.Changelog[enabledModule.String()]; ok {
+			enabledModulesChangelog[enabledModule.String()] = v
+		}
+	}
+
+	// enable global modules
+	if v, ok := dcr.releaseMetadata.Changelog["global"]; ok {
+		enabledModulesChangelog["global"] = v
```

Comment:
There is a tip about possible values in CHANGELOG.
https://github.com/deckhouse/deckhouse/blob/4dc5d5bd0c67ba17218a728db03b3524af39084c/.github/pull_request_template.md?plain=1#L35-L49

What do you think about:
* Filtering all low impact changes?
* Also, always add deckhouse-controller and dhctl sections to the CR?

Reply:
We agreed that @yalosev would add two sections: candi and deckhouse-controller. No severity filter is required.